app/scripts/arbeitseinsaetze/list/arbeitseinsaetzelist.html

```diff
@@ -34,8 +34,14 @@
                 </td>
                 <td>
                   <div class="btn-group">
-                    <a role="button" class="btn btn-sm btn-warning" ng-click="quit(arbeitseinsatz)"><span class="glyphicon glyphicon-minus"></span></a>
-                    <a role="button" class="btn btn-sm btn-info" ng-click="edit(arbeitseinsatz)"><span class="glyphicon glyphicon-edit"></span></a>
+                    <script type="text/ng-template" id="workAssignmentInfoTooltipTemplate.html">
+                      <span translate> 
+                        Sie können diese Zuordnung nicht entfernen, da sie zu nahe am Ausführungsdatum liegt
+                      </span>
+                    </script>
+                    <a ng-if="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion)" ng-disabled="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion)" role="button" uib-tooltip-template="'workAssignmentInfoTooltipTemplate.html'" class="btn btn-sm btn-warning ng-scope" ng-click="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion) || quit(arbeitseinsatz)"><span class="glyphicon glyphicon-minus"></span></a>
```

Comment:
ng-if and ng-disabled is the same. Shouldn't it be on ng-disabled only? And then the uib-tooltip-template should only display the above message, if the button is disabled, right? This could be acheived on the template.